docs/samples/deployments/spark/index.md

```diff
@@ -0,0 +1,148 @@
+# Anonymize PII using Presidio on Spark
+
+You can leverages presidio to perform data anonymization as part of spark notebooks.
+The following sample uses [Azure Databricks](https://docs.microsoft.com/en-us/azure/databricks/) and simple text files hosted on [Azure Blob Storage](https://docs.microsoft.com/en-us/azure/storage/blobs/). However, it can easily change to fit any other scenario which requires PII analysis or anonymization as part of spark jobs.
+
+## Pre-requisites
+
+Provision and setup the required infrastrucutre
+
+### Azure Storage
+
+Provision an Azure storage account using the following script.
+
+``` bash
+RESOURCE_GROUP=[resource group name]
+STORAGE_ACCOUNT_NAME=[storage account name]
+STORAGE_CONTAINER_NAME=[blob container name]
+DATABRICKS_WORKSPACE_NAME=[databricks workspace name]
+DATABRICKS_SKU=[basic/standard/premium]
+LOCATION=[location]
+
+# Create the storage account
+az group create --name $RESOURCE_GROUP --location $LOCATION
+az storage account create --name $STORAGE_ACCOUNT_NAME --resource-group
+$RESOURCE_GROUP
+
+# Get the storage account access key
+STORAGE_ACCESS_KEY=$(az storage account keys list --account-name $STORAGE_ACCOUNT_NAME --resource-group $RESOURCE_GROUP --query '[0].value' -o tsv)
+
+# Create a blob container
+az storage container create -n $STORAGE_CONTAINER_NAME --account-name $STORAGE_ACCOUNT_NAME
+```
+
+### Azure Databricks
+
+Provision an Azure databricks service using the following script.
+
+```bash
+az extension add --name databricks
+
+az databricks workspace create --resource-group $RESOURCE_GROUP  --name $DATABRICKS_WORKSPACE_NAME --sku $DATABRICKS_SKU
+```
+
+Initialze a cluster, either using the [Azure databricks UI](https://docs.microsoft.com/en-us/azure/databricks/clusters/create) or by following up the next script using [databricks cli](https://docs.microsoft.com/en-us/azure/databricks/clusters/create).
+
+```bash
+# Configure databricks token. for host name use https://[databricks region name].azuredatabricks.net. for token acquire a PAT using the following guide: https://docs.microsoft.com/en-us/azure/databricks/dev-tools/api/latest/authentication#--generate-a-personal-access-token
+# 
+databricks configure --token
+
+databricks clusters create --json "{ 
+    \"cluster_name\": \"presidio\",
+    \"spark_version\": \"7.5.x-scala2.12\", 
+    \"autoscale\": {
+        \"min_workers\": 2,
+        \"max_workers\": 8
+    },
+    \"azure_attributes\": { 
+        \"first_on_demand\": 1, 
+        \"availability\": \"ON_DEMAND_AZURE\", 
+        \"spot_bid_max_price\": -1 
+    }, 
+    \"node_type_id\": \"Standard_F4s_v2\", 
```

Comment:
Did we have a specific reason for choosing the `Standard_F4s_v2` SKU? If so we could mention it in this readme, or we could leave it to the General purpose D-series VM sizes.

Reply:
done